src/app/pages/FrontPage/FrontPage.jsx

```diff
@@ -111,7 +111,7 @@ const FrontPage = ({ pageData, mostReadEndpointOverride }) => {
   return (
     <>
       {/* dotcom and dotcomConfig need to be setup before the main dotcom javascript file is loaded */}
-      {hasAds && !isAmp && <CanonicalAdBootstrapJs />}
+      {hasAds && canAdvertise && !isAmp && <CanonicalAdBootstrapJs />}
```

Comment:
Can we make sure we link to some notes in a readme about this? Looking at the code, it can be difficult to know what the conceptual difference is between `hasAds` and `canAdvertise`, without stepping through the full application.

Reply:
Or we could rename the prop to a more meaningful name like `showAdsBasedOnLocation`.